examples/models/file/calendars.py

```diff
@@ -1,3 +1,16 @@
+''' A rendering of the 2014 monthly calender.
+
+This example demonstrates the usage of plotting several
+plots together using ``gridplot``.
+
+A hover tooltip displays the US holidays on the significant dates.
+
+.. bokeh-example-metadata::
+    :sampledata: us_holidays
+    :apis: bokeh.layouts.gridplot, bokeh.models.tools.HoverTool, bokeh.models.plots.Plot, bokeh.models.glyphs.Rect, bokeh.models.glyphs.Text, bokeh.document.document.Document # noqa: E501
+    :refs: :ref:`userguide_layout` > :ref:`userguide_layout_gridplot`, :ref:`userguide_tools` > :ref:`userguide_tools_hover_tool`
+    :keywords: gridplot, hover, tooltip
+'''
```

Comment:
We typically leave a blank line at the bottom of every docstring, though I am not sure this documented anywhere (cc @tcmetzger)
```suggestion
    :keywords: gridplot, hover, tooltip

'''
```